src/index.js

```diff
@@ -847,30 +864,46 @@ class MiniCssExtractPlugin {
               return null;
             }
 
+            const attributes = {
+              href: `${RuntimeGlobals.publicPath} + ${RuntimeGlobals.require}.miniCssF(chunkId)`,
+              rel: JSON.stringify('stylesheet'),
+              onload: 'onLinkComplete',
+              onerror: 'onLinkComplete',
+            };
+
+            // Some attributes cannot be assigned through setAttribute, so we maintain
+            // a list of attributes that can safely be assigned through dot notation
+            const safeAttrs = ['href', 'rel', 'type', 'onload', 'onerror'];
```

Comment:
I think we should change this logic little bit, I think we don't protect attributes, developer can add `data-*` attributes, any non valid HTML attribute (why not :smile: ), even onload and onerror can be rewritten

Reply:
The reason for this logic is that some attributes must be assigned through `tag.prop = value` while others must be assigned through `tag.setAttribute("prop", value);`. The array above is a bit over-cautious, as I found that only `onload` and `onerror` needed to be set through dot notation (e.g. `tag.onload = ...`), the other attributes defined there can just as easily be set through `setAttribute`.

I'm not aware of any sensible way of determining what attributes needs to be set which way, so one proposal here would be to rewrite the above code to say
```
const safeAttrs = ['onload', 'onerror'];
```

Perhaps we could also update the wording and the comment to clarify why this is done.

The problem with that solution is that user-defined attributes (other than onload and onerror) that must be assigned through dot notation will not work with that solution. What's your thoughts on this?

Reply:
Hi @alexander-akait ,
hope you have nice summer weather =)
Could you please prioritise it a bit? Currently it's annoying as we have to use preload/prefetch with SRI, but chrome/safary/ff are not happy to preload with SRI but then load it without SRI.

Reply:
@utlime need rebase, I think it is stale PR, so feel free to send this code again and we will merge it

Reply:
@alexander-akait Not at all, I have been waiting for your response to my latest comment

Reply:
I think we can write warning in docs about some attributes, they are still internal and should not change, it will simplify logic